pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
A-ha, I presume that this constraint is because of the surrounding code? I judged it only by this line alone, sorry.

Then we can just replace this whole line by `[getBin out]`, right? (just to confirm, because from just looking at the line this changes the logic and I don't know the context good enough)